### PR TITLE
Feature/#2597 update last activity time

### PR DESCRIPTION
**Pull request by paslavskyi9roman**
This PR contains the implementation of forming and sending to the backend the date when the user leaves the page

Original task: https://github.com/ita-social-projects/GreenCity/issues/2597

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "fileuploader": "^1.2.7",
     "firebase": "^7.20.0",
     "hammerjs": "^2.0.8",
+    "moment": "^2.29.1",
```
Comment (JimHolman1199) [inline package.json:51]: we don't need to set the dependency for 1 use only. so please consider using native js.

File: src/app/service/user/user.service.ts [hunk 2]
```diff
@@ -223,6 +224,12 @@ export class UserService implements OnLogout {
     });
   }
 
+  updateLastTimeActivity() {
+    const date = new Date();
+    const currentDate = moment(date).format('yyyy-MM-DDTHH:mm:ss.SSSSSS');
```
Comment (JimHolman1199) [inline src/app/service/user/user.service.ts:229]: please move this line to a separate method.

File: package.json (hunk 1 above)
Comment (paslavskyi9roman) [reply package.json:51]: we already have moment.js, but older version. I've just updated the existing one. Here is the screen from dev branch:
image

Comment (JimHolman1199) [reply package.json:51]: this screen from package-lock.json?

Comment (paslavskyi9roman) [reply package.json:51]: Yes, indeed

File: src/app/service/user/user.service.ts (hunk 2 above)
Comment (paslavskyi9roman) [reply src/app/service/user/user.service.ts]: A separete reusable method for date converting has been created with this commit a535799dc3be146d73c955ca6b0c6f40dddd115c

**Review by idzhersh — APPROVED** [review]

**Review by KirillAbramo — APPROVED** [review]

**Review by Edw1x — APPROVED** [review]

**Review by yuriiLisunov — APPROVED** [review]

**Review by AlekOK — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**